includes/admin/events/metaboxes.php

```diff
@@ -624,6 +625,14 @@ function mdjm_event_metabox_options_playlist_row( $event_id )	{
 			'current'  => $enable_playlist ? 'Y' : 0,
 		) ); ?> <?php _e( 'Enable Playlist?', 'mobile-dj-manager' ); ?></p>
 
+    <p><?php _e ( 'Playlist Limit', 'mobile-dj-manager' ); ?>
+
+    <?php echo MDJM()->html->text( array(
```

Comment:
Can this be a number field?

Again, check alignment.

    <p><?php echo MDJM()->html->text( array(
          'name'  => '_mdjm_event_playlist_limit',
          'class' => '',
          'value' => ! empty( $playlist_limit ) ? $playlist_limit : ''
    ) ); ?></p>